Common/Model.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.Collections.Generic;
+using OpenTK.Graphics.OpenGL4;
+using OpenTK.Mathematics;
+using Assimp;
+using System.Drawing;
+using System.Drawing.Imaging;
+using PixelFormat = OpenTK.Graphics.OpenGL4.PixelFormat;
+using TextureWrapMode = OpenTK.Graphics.OpenGL4.TextureWrapMode;
+using AssimpMesh = Assimp.Mesh;
+
+namespace LearnOpenTK.Common
+{
+    public class Model
+    {
+        // model data
+        private List<Mesh> meshes;
+
+        // constructor, expects a filepath to a 3D model.
+        public Model(string path)
+        {
+            LoadModel(path);
+        }
+
+        // draws the model, and thus all its meshes
+        public void Draw()
+        {
+            for (int i = 0; i < meshes.Count; i++)
+                meshes[i].Draw();
+        }
+
+        // loads a model with supported ASSIMP extensions from file and stores the resulting meshes in the meshes list.
+
+        private void LoadModel(string path)
+        {
+            //Create a new importer
+            AssimpContext importer = new AssimpContext();
+
+            //This is how we add a logging callback 
```

Comment:
This comes directly from the [Assimp-Net Wiki](https://code.google.com/archive/p/assimp-net/wikis/GettingStarted.wiki) and from what I understand, it attaches a function to the importer so that during the import you can see the steps that are happening and they will be written to the console. The documentation for AssimpNet is rather lacking (I couldn't get their [chm file](https://github.com/assimp/assimp-net/blob/master/Docs/AssimpNet%20Documentation%20(July%202014).chm) to work. I am adding some comments to explain it better, since looking more into it, I think it is a very useful debugging tool.

Reply:
Are you interested in expanding on this comment to explain it a bit more? Should be the last thing before I can merge it.

Reply:
I made some changes to the comments which hopefully clarify this better.